### DISCUSSION TITLE
Testing in the editor now doesn't automatically end when running out of objects

### DISCUSSION THREAD

**Discussion by snalgae** [ppy/osu#28236] [Q&A] [answered]
When pressing test and going to the end of all notes, 
previously it would bring you back to the editor, 
but now you have to manually press pause and quit to go back to the editor.
This affects all modes
I'm not sure whether to consider this a bug, so I'm posting it here.

I think It would be preferable for the transition between testing and editor to take only a single key press

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/17395
https://github.com/ppy/osu/issues/23029
